gradle/libs.versions.toml

```diff
@@ -32,6 +32,8 @@ app-icon-loader-coil = "1.5.0"
 skrapeit = "1.2.2"
 libsu = "5.2.2"
 scrollbars = "1.0.4"
+enumutil = "1.1.0"
+enumutil-ksp = "1.1.0"
```

Comment:
Defining two versions is unnecessary unless `enumutil` and `enumutil-ksp` are actually versioned separately